src/pytest/test_aggregate.py

```diff
@@ -470,3 +470,21 @@ def testGroupbyNoReduce(env):
     for row in rv[1:]:
         env.assertEqual('primaryName', row[0])
         env.assertTrue('sarah' in row[1])
+
+from hotels import hotels
+def testGeo(env):
+    env.expect('ft.create', 'idx', 'schema', 'name', 'text', 'location', 'geo').ok()
+
+    for i, hotel in enumerate(hotels):
+        env.expect('ft.add', 'idx', 'hotel{}'.format(i), 1.0, 'fields', 'name',
+                    hotel[0], 'location', '{},{}'.format(hotel[2], hotel[1])).ok()
+
+    res = [102L, ['distance', '0'], ['distance', '95.43'], ['distance', '399.66'], ['distance', '1896.44'],
+                 ['distance', '2018.14'], ['distance', '2073.48'], ['distance', '2640.42'],
+                 ['distance', '2715.46'], ['distance', '3657.74'], ['distance', '4047.96']]
+
+    env.expect('ft.aggregate', 'idx', 'hilton',
+               'LOAD', '1', '@location',
+               'APPLY', 'distance(@location,-0.15036,51.50566)', 'AS', 'distance',
```

Comment:
I think its better that the function will get an equivalent arguments type and not (string, num, num). So I think something like distance(string, string) is better (we will try to parse both string as geo). This way we can also check the distance between two fields in the document for example..

Reply:
I wanted it this way but the parser automatically treats `','` as separator and therefore splits the string.
We can have another option that will receive two field names.
I should probably add an option for type (meter, feet...). This will make the pressing annoying (combined with the above options). Any idea how to streamline this?

Reply:
You mean if I write something like this : `func("a,b")` it gets it as 2 arguments? This is a bad by itself..

Reply:
in agg/expr/parser.y
```
arglist(A) ::= . [ARGLIST] { A = RS_NewArgList(NULL); }
arglist(A) ::= expr(B) . [ARGLIST] { A = RS_NewArgList(B); }
arglist(A) ::= arglist(B) COMMA expr(C) . [ARGLIST] { 
    A = RSArgList_Append(B, C);
}
```
We use it to split args
https://oss.redislabs.com/redisearch/Aggregations/#list_of_string_apply_functions

Reply:
What do we decide about this? I do not think we should @gkorland @K-Jo? I do think the api should get 2 strings and not string,num,num

Reply:
@MeirShpilraien what do you mean by two strings?

Reply:
Currently, the parser splits arguments with `,` and therefore, an input `distance(@field,1.2,-3.4)` will be parsed as `(string, num, num)`. 
We can decide on another char for splitting between long and lat but it won't be intuitive.